ui/package.json

```diff
@@ -59,6 +59,7 @@
     "react-router": "^6.2.1",
     "react-router-bootstrap": "^0.26.1",
     "react-router-dom": "^6.19.0",
+    "react-select": "5.8.0",
```

Comment:
Hey @meguiraun, you should use Bootstrap's `Form.Select` component instead. This is what is used for other drop-down selectors. For instance, check out the code for the "Beam size" selector (the component is called `ApertureInput`).

Reply:
mmm.... I cannot customize the style of the different options. 

Based on the currently selected beam focus, the options for the aperture will be displayed with a different color or even disabled.

react-select allows me this customization, but I am struggling with Form.Select
<img width="781" alt="image" src="https://github.com/mxcube/mxcubeweb/assets/12758327/47513ea2-1e66-475b-a6a9-3f96bfd3f4fa">

Reply:
you are my hero 🤗